manifests/l/lars-berger/GlazeWM/1.9.0/lars-berger.GlazeWM.installer.yaml

```
@@ -0,0 +1,17 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: lars-berger.GlazeWM
+PackageVersion: 1.9.0

```

```suggestion
PackageVersion: 1.9.0
Commands:
- glazewm
```
To allow the application to run from the terminal with `glazewm` command.